### ISSUE TITLE
Add test cases to continuous integration

### ISSUE THREAD

**Issue by gaocegege** [coala/coala-vs-code#6]

**Comment by gaocegege** [comment]
There are three types of test cases needed.

1. Server side unit test
2. Client side unit test
3. Integration test supported by vs code.

### reference

[Testing Your Extension](https://code.visualstudio.com/Docs/extensions/testing-extensions)

**Comment by gaocegege** [comment]
Bad GitHub.

reopen it because client side test is not ready.

**Comment by jayvdb** [comment]
So this needs to be uncommented: https://github.com/coala/coala-vs-code/blob/master/.travis.yml#L57 .. and errors fixed .. ?

**Comment by gaocegege** [comment]
No, we have no test for the client, just server side test cases exists. Thus we should write tests for client.

I am not sure if we could do unit test for the client. If not we should add e2e test, maybe.

**Comment by jayvdb** [comment]
ok, changed this to be just about the client.
